mnd/mnd/registry/patients/mnd_admin_forms.py

```diff
@@ -154,10 +148,6 @@ def set_patient(self, patient):
     def clean_email(self):
         email = self.cleaned_data['email']
         instance = getattr(self, 'instance', None)
-        if self.carer_has_user():
```

Comment:
Should delete `carer_has_user` if it's no longer used, or use it in the `save`

Reply:
Updated.